### ISSUE TITLE
Add 'Our stories' as new item in the main menu

### ISSUE THREAD

**Issue by micheleminno** [humanetech-community/community-hub#52]
I'll remove it from resources and place in a dedicated page accessible from the main menu.

Do we have concerns about publishing member introductions here?

Both site (forum and website) are public, so introductions will be accessible anyway, but @gkrishnaks and @patmatsu told that it's a privacy violation.

**Comment by micheleminno** [comment]
From @gkrishnaks: I think it’s a good idea to ask individual person for permission, or just inform everyone that any post in introduction section goes to the site as both are public.

From @patmatsu: I agree. There should be a notice somewhere that information can be shared at the discretion of the admins.

**Comment by aschrijver** [comment]
Technically it should be no problem, as the texts are CC-BY and attribution can be made, but I agree with the above. It is uncivil to just republish elsewhere. These people introduced themselves to the other community members, not to the outside world. Therefore permission is required.

I would like to propose foregoing the 'Our stories' idea, and instead turn this into a 'People Directory'. I'll follow up on the forum to elaborate.

**Comment by aschrijver** [comment]
I don't think we should have 'Our stories' in the main menu. If there is something to be added there I'd rather include a 'People' entry to the main menu that leads to the People directory where people can introduce themselves and explain their interests regarding Humane Technology.

The People directory is a Who's Who lookup for people to find others to cooperate with. This is an often asked question ("Do you know researchers that can help me with [this and that]?")

Proposing to close this issue.

**Comment by patmatsu** [comment]
OK with me to close.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by micheleminno** [comment]
According to me, an 'Our stories' page would be better that a 'People directory', or if we want we can have both. The idea behind 'Our stories' is this:

-  having a place where new members can feel themselves more in connection to the existing members of the community, in a way that they can find even naive similarities with some of them and experience some sort of shared feeling. They need to bend those humane tech big words and objectives down to their ground of normal people, addicted students, concerned parents, unsatisfied coders, etc. 

The idea behind 'People directory' (I suppose) is this:

- more professionals-oriented, with people listed with their own profession, field of research, interests related to the humane tech world.

For me the first objective is more important, because people need to be encouraged to share their stories in terms of their relation with current technology and see other similar stories they can relate to. A people directory is more useful among people already working or studying in this field, it could be inside the 'Be a builder' page.

**Comment by aschrijver** [comment]
> According to me, an 'Our stories' page would be better that a 'People directory', or if we want we can have both.

Yes, you are right. My current idea is to have a 'People' main menu entry that leads to a page where you have a combined 'Our stories' and 'People directory'. Personally I think the people directory is more important than the stories. Most important goal is to get members actively contributing, and 'feeling at home' is only where this starts. 

I kindly disagree on your last paragraph. The forum offers enough engagement for people to feel at home. The community itself should not be a cozy discussion corner, but for 'bulders' and 'activists' - do-ers that roll up their sleeves and get to work.

Right now the problem is not that people do not feel at home. The issue is to get more people to become active and to stay active.

So let's have both Stories and Directory below the 'People' main menu entry..

**Comment by micheleminno** [comment]
> I kindly disagree on your last paragraph. The forum offers enough engagement for people to feel at home. The community itself should not be a cozy discussion corner, but for 'bulders' and 'activists' - do-ers that roll up their sleeves and get to work.

It's not about 'feeling at home'. It's about letting each person find out why she/he is here, share it and find deep reasons to be motivated and escape from that 'I'm the only one feeling it' sensation that many times occurs when you are feeling something (it could be about gender, sex, political, social or technology use and consequences issues, like in our case) that all other people you know don't feel, or seem not to feel.
This should be both the precondition to start to actively work and the key to agree about what to work on, naturally splitting into groups of people that are _empathically_, more than technically, similar one to the other.

**Comment by aschrijver** [comment]
Okay, let's have both as I proposed. I am feeling the need for people directory most pressingly, as I get PM'ed a lot with questions like "Who can I approach for [this and that]" asking for specific professions and it is time-consuming, and I don't have a good answer.
